test/include/RAJA_test-launch-execpol.hpp

```diff
@@ -28,7 +28,7 @@ using Sequential_launch_policies = camp::list<
 #if defined(RAJA_ENABLE_OPENMP)
 using omp_policies = camp::list<
          RAJA::LaunchPolicy<RAJA::omp_launch_t>,
-         RAJA::LoopPolicy<RAJA::omp_parallel_for_exec>  
+         RAJA::LoopPolicy<RAJA::omp_for_exec>
```

Comment:
Are we using `omp_for_exec` because the parallel region is instantiated in the top level launch?

Reply:
yes, omp_launch_t opens up the parallel region.